tests/v2/validation/provisioning/config.go

```diff
@@ -0,0 +1,133 @@
+package provisioning
```

Comment:
why did this move?

Reply:
Hmm, something got lost in translation when I was rebasing off the refactor. I'll fix this.

Reply:
Resolved.